documentation/modules/mirrormaker2/proc-mirrormaker-replication.adoc

```diff
@@ -0,0 +1,104 @@
+
+.Prerequisites
+
+* xref:cluster-operator-str[{ProductName} and Kafka is deployed]
+* Source and target Kafka clusters are available
+
+.Procedure
+
+. Edit the `spec` properties for the `KafkaMirrorMaker2` resource.
++
+The properties you can configure are shown in this example configuration:
++
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: KafkaMirrorMaker2
+metadata:
+  name: my-mirror-maker2
+spec:
+  version: {DefaultKafkaVersion} <1>
+  replicas: 3 <2>
+  connectCluster: "my-cluster-target" <3>
+  clusters: <4>
+  - alias: "my-cluster-source" <5>
+    bootstrapServers: my-cluster-source-kafka-bootstrap:9092 <6>
+  - alias: "my-cluster-target" <7>
+    bootstrapServers: my-cluster-target-kafka-bootstrap:9092 <8>
+    config: <9>
+      config.storage.replication.factor: 1
+      offset.storage.replication.factor: 1
+      status.storage.replication.factor: 1
+  mirrors: <10>
+  - sourceCluster: "my-cluster-source" <11>
+    targetCluster: "my-cluster-target" <12>
+    sourceConnector: <13>
+      config:
+        replication.factor: 1 <14>
+        offset-syncs.topic.replication.factor: 1 <15>
+        sync.topic.acls.enabled: "false" <16>
+    heartbeatConnector: <17>
+      config:
+        heartbeats.topic.replication.factor: 1 <18>
+    checkpointConnector: <19>
+      config:
+        checkpoints.topic.replication.factor: 1 <20>
+    topicsPattern: ".*" <21>
+    groupsPattern: "group1|group2|group3" <22>
+----
+<1> The Kafka Connect version.
+<2> The number of replica nodes.
+<3> The cluster alias for Kafka Connect.
+<4> Specification for the Kafka clusters being synchronized.
+<5> The cluster alias for the source Kafka cluster.
+<6> Bootstrap server for connection to the source Kafka cluster.
+<7> The cluster alias for the target Kafka cluster.
+<8> Bootstrap server for connection to the target Kafka cluster.
+<9> xref:assembly-kafka-connect-configuration-deployment-configuration-kafka-connect[Kafka Connect configuration].
+Standard Apache Kafka configuration may be provided, restricted to those properties not managed directly by {ProductName}.
+<10> MirrorMaker 2.0 connectors.
+<11> The alias of the source cluster used by the MirrorMaker 2.0 connectors.
+<12> The alias of the target cluster used by the MirrorMaker 2.0 connectors.
+<13> The configuration for the `sourceConnector` that creates remote topics.
+<14> The replication factor for the `sourceConnector`.
+<15> The replication factor for the `sourceConnector` `offset-syncs` internal topic that maps the offsets of the source and target clusters.
+<16> When enabled, ACLs are applied to synchronized topics. The default is `false`.
+<17> The configuration for the `heartbeatConnector` that performs connectivity checks.
+<18> The replication factor for the `heartbeatConnector`.
+<19> The configuration for the `checkpointConnector` that tracks offsets.
+<20> The replication factor for the `checkpointConnector`.
```

Comment:
This option sets the replication factor for the checkpoint topic that is created by the `MirrorCheckpointConnector`, something like:
```
<14> The replication factor for the checkpoints topic created at the target cluster.
```